addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/onboard-account.cfn.yml

```diff
@@ -751,14 +751,7 @@ Resources:
             Principal:
               AWS: '*'
             Action:
-              - 'kms:DescribeKey'
-              - 'kms:Encrypt'
-              - 'kms:Decrypt'
-              - 'kms:ReEncrypt*'
-              - 'kms:GenerateDataKey'
-              - 'kms:GenerateDataKeyWithoutPlaintext'
-              - 'kms:CreateGrant'
-              - 'kms:RevokeGrant'
+              - 'kms:*'
```

Comment:
why changing to all actions?

Reply:
The cloudformation was failed before changed to 'kms:*', below is error message:
`The new key policy will not allow you to update the key policy in the future`

But I deep dived again, I found this is caused by my [commit](https://github.com/awslabs/service-workbench-on-aws-cn/commit/01e6ce53a4b1e12c7108b0aa7b3a2834acc9faf7) , which change AWS::AccountId to CentralAccountId, this caused the accounts are different when we add the external aws account.  
I changed it back. 
Thank you very much to help find this issue.